lint/file.go

```diff
@@ -24,6 +24,23 @@ type File struct {
 // IsTest returns if the file contains tests.
 func (f *File) IsTest() bool { return strings.HasSuffix(f.Name, "_test.go") }
 
+// IsImportable returns true if the symbols defined in this file can be imported in other packages.
+// If the file is in the main package, it is not importable.
+// If the file is a test file, it is not importable.
+func (f *File) IsImportable() bool {
+	if f.Pkg.IsMain() {
+		// The main package is not importable.
+		return false
+	}
+
+	if f.IsTest() {
+		// The test files are not importable.
+		return false
+	}
+
+	return true
```

Comment:
minor, but we can keep it simple:
```suggestion
	return !f.Pkg.IsMain() && !f.IsTest()
```